demisto_sdk/__main__.py

```diff
@@ -943,6 +951,7 @@ def upload(**kwargs):
         kwargs.pop('zip')
         kwargs.pop('keep_zip')
         kwargs.pop('input_config_file')
+        kwargs.pop('xsiam')
```

Comment:
@barchen1 Is it possible that `xsiam` will be popped more than once? If so, just add a `None` default value to the pop so it won't raise an exception.